iceoryx_hoofs/include/iceoryx_hoofs/posix_wrapper/signal_handler.hpp

```diff
@@ -69,7 +69,9 @@ class SignalGuard
 
   private:
     Signal m_signal;
-    struct sigaction m_previousAction;
+    struct sigaction m_previousAction
```

Comment:
Please write `sigaction m_previousAction{};` to get rid of the formatting issue.